packages/vue/src/components/molecules/SfImage/SfImage.html

```diff
@@ -1,17 +1,13 @@
-<div
-  class="sf-image"
-  :style="{maxWidth: maxWidth}"
-  @mouseover="hoverHandler(true)"
-  @mouseleave="hoverHandler(false)">
+<div class="sf-image" :style="{maxWidth: maxWidth}" @mouseover="hoverHandler(true)" @mouseleave="hoverHandler(false)">
   <transition :name="transition">
     <div v-if="hasOverlay && loaded" class="sf-image__overlay">
       <!--@slot slot for overlay content, if is empty the overlay is off -->
-      <slot/>
+      <slot />
     </div>
   </transition>
-  <img
-    ref="img"
-    loading="lazy"
-    class="sf-image__img"
-  >
+  <img ref="img" loading="lazy" class="sf-image__img lozad" v-if="src" />
```

Comment:
não precisa mais do loadind=lazy porque vamos suar lozad, e a classe lozad também não é necessária porque você está criando o observer pela classe sf-image__img, certo?

```suggestion
  <img :data-src="src" class="sf-image__img" v-if="src" />
```